repo_policy_compliance/__init__.py

```diff
@@ -175,14 +214,22 @@ def all_(input_: Input, policy_document: dict | UsedPolicy = UsedPolicy.ALL) ->
         return source_branch_report
 
     if (
-        policy.enabled(name=policy.Property.COLLABORATORS, policy_document=used_policy_document)
+        policy.enabled(
+            job_type=policy.JobType.PULL_REQUEST,
+            name=policy.PullRequestProperty.COLLABORATORS,
+            policy_document=used_policy_document,
+        )
```

Comment:
Could this list of ifs benefit from a match-case structure instead? https://peps.python.org/pep-0636/

Reply:
Those work quite well if there are different paths depending on a single value. In this case, each of the if statement needs to retrieve its own report and return a report depending on that returned report so a pattern matching isn't well suited for that